content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -408,6 +418,39 @@ You can configure this behavior for the `spec` of a Pod using a
 [projected volume](/docs/concepts/storage/volumes/#projected) type called
 `ServiceAccountToken`.
 
+The token from this projected volume is a JSON Web Token (JWT).  The JSON payload of this
+token follows a well defined schema - an example payload for a pod bound token:
+
+```json
+{
+  "aud": [
+    "https://kubernetes.default.svc"
```

Comment:
this audience is also configurable, right? probably worth calling that out or people will hard-code stuff based on this doc

Reply:
Done.